packages/amplify-category-storage/commands/storage/add.js

```diff
@@ -26,10 +26,13 @@ module.exports = {
         const { print } = context;
         print.success(`Successfully added resource ${resourceName} locally`);
         print.info('');
+        print.warning(
+          'If a user is part of a user pool group, run "amplify update storage" to enable IAM group policies for CRUD operations',
```

Comment:
nit: remove ,

Reply:
These comma's get added by prettier in our commit hooks I believe